debezium-storage/debezium-storage-rocketmq/src/main/java/io/debezium/storage/rocketmq/history/RocketMqSchemaHistory.java

```diff
@@ -0,0 +1,313 @@
+                .aclEnable(config.getBoolean(RocketMqSchemaHistory.ROCKETMQ_ACL_ENABLE))
+                .accessKey(config.getString(RocketMqSchemaHistory.ROCKETMQ_ACCESS_KEY))
+                .secretKey(config.getString(RocketMqSchemaHistory.ROCKETMQ_SECRET_KEY))
+                .namesrvAddr(config.getString(RocketMqSchemaHistory.NAME_SRV_ADDR))
+                .groupId(dbHistoryName)
+                .build();
+    }
+
+    @Override
+    public void initializeStorage() {
+        super.initializeStorage();
+        LOGGER.info("try to create history topic: {}!", this.topicName);
+        TopicConfig topicConfig = new TopicConfig(this.topicName, 1, 1, 6);
+        RocketMqAdminUtil.createTopic(rocketMqConfig, topicConfig);
+    }
+
+    @Override
+    public synchronized void start() {
+        super.start();
+        try {
+            // Check and create group
+            Set<String> consumerGroupSet = RocketMqAdminUtil.fetchAllConsumerGroup(rocketMqConfig);
+            if (!consumerGroupSet.contains(rocketMqConfig.getGroupId())) {
+                RocketMqAdminUtil.createSubGroup(rocketMqConfig, rocketMqConfig.getGroupId());
+            }
+            // Start rocketmq producer
+            this.producer = RocketMqAdminUtil.initDefaultMQProducer(rocketMqConfig);
+            this.producer.start();
+        }
+        catch (MQClientException e) {
+            throw new SchemaHistoryException(e);
+        }
+    }
+
+    @Override
+    protected void storeRecord(HistoryRecord record) throws SchemaHistoryException {
+        if (this.producer == null) {
+            throw new IllegalStateException("No producer is available. Ensure that 'initializeStorage()'"
+                    + " is called before storing database schema history records.");
+        }
+
+        LOGGER.trace("Storing record into database schema history: {}", record);
+        try {
+            Message message = new Message(this.topicName, record.toString().getBytes());
+            producer.send(message, new ZeroMessageQueueSelector(), null, new SendCallback() {
+                @Override
+                public void onSuccess(SendResult sendResult) {
+                    LOGGER.debug("Stored record in topic '{}' partition {} at offset {} ",
+                            message.getTopic(), sendResult.getMessageQueue(), sendResult.getMessageQueue());
+                }
+
+                @Override
+                public void onException(Throwable e) {
+                    LOGGER.error("Store record into database schema history failed : {}", e);
+                }
+            });
+        }
+        catch (InterruptedException e) {
+            LOGGER.error("Interrupted before record was written into database schema history: {}", record);
+            Thread.currentThread().interrupt();
+            throw new SchemaHistoryException(e);
+        }
+        catch (MQClientException | RemotingException e) {
+            throw new SchemaHistoryException(e);
+        }
```

Comment:
this will return without being sure the event was successfully written to the topic - we should wait here for the results if it was written or not, e.g. by `latch.await()`, similar to RockerMQ sink adapter.